src/service-object.ts

```diff
@@ -409,11 +415,11 @@ class ServiceObject extends EventEmitter {
 
     reqOpts.interceptors_ = childInterceptors.concat(localInterceptors);
 
-    if (!callback) {
+    if (returnStream) {
```

Comment:
And one more thought-- to be more idiomatic, can it either be part of `reqOpts` or another object for configuration? Seeing `this.request_(reqOpts, true)` is a bit too Java-- even though smart IDEs will get it, it can be troubling for GH-browsing.

Reply:
> is a bit too Java

Hey!  I'm a C# developer thank you.